packages/p2p/src/components/buy-sell/buy-sell-modal.jsx

```diff
@@ -0,0 +1,156 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { Button, MobileFullPageModal, Modal, ThemedScrollbars } from '@deriv/components';
+import { isMobile, useIsMounted } from '@deriv/shared';
+import { observer } from 'mobx-react-lite';
+import { buy_sell } from 'Constants/buy-sell';
+import { localize } from 'Components/i18next';
+import FormError from 'Components/form/error.jsx';
+import { useStores } from 'Stores';
+import BuySellForm from './buy-sell-form.jsx';
+import NicknameForm from '../nickname/nickname-form.jsx';
+import 'Components/buy-sell/buy-sell-modal.scss';
+
+const BuySellModalFooter = ({ onCancelClick, error_message, is_submit_disabled, onSubmit }) => {
```

Comment:
How about `onCancel` instead of `onCancelClick`, similar to `onSubmit` ?

Reply:
This is done.